bigbio/hub/hub_repos/jnlpba/jnlpba.py

```diff
@@ -13,68 +13,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-The data came from the GENIA version 3.02 corpus (Kim et al., 2003).
-This was formed from a controlled search on MEDLINE using the MeSH terms human, blood cells and transcription factors.
-From this search 2,000 abstracts were selected and hand annotated according to a small taxonomy of 48 classes based on
-a chemical classification. Among the classes, 36 terminal classes were used to annotate the GENIA corpus.
-"""
-
+import itertools
+import os
 from typing import Dict, List, Tuple
 
 import datasets
 
-from .bigbiohub import kb_features
-from .bigbiohub import BigBioConfig
-from .bigbiohub import Tasks
+from .bigbiohub import BigBioConfig, Tasks, kb_features
 
-_LANGUAGES = ['English']
+_LANGUAGES = ["English"]
 _PUBMED = True
 _LOCAL = False
 
-# TODO: Add BibTeX citation
 _CITATION = """\
-@inproceedings{collier-kim-2004-introduction,
-title = "Introduction to the Bio-entity Recognition Task at {JNLPBA}",
-author = "Collier, Nigel and Kim, Jin-Dong",
-booktitle = "Proceedings of the International Joint Workshop
-on Natural Language Processing in Biomedicine and its Applications
-({NLPBA}/{B}io{NLP})",
-month = aug # " 28th and 29th", year = "2004",
-address = "Geneva, Switzerland",
-publisher = "COLING",
-url = "https://aclanthology.org/W04-1213",
-pages = "73--78",
+@inproceedings{collier2004introduction,
+title={Introduction to the bio-entity recognition task at JNLPBA},
+author={Collier, Nigel and Ohta, Tomoko and Tsuruoka, Yoshimasa and Tateisi, Yuka and Kim, Jin-Dong},
+booktitle={Proceedings of the International Joint Workshop on Natural Language Processing in Biomedicine \
+ and its Applications (NLPBA/BioNLP)},
+pages={73--78},
+year={2004}
 }
 """
 
 _DATASETNAME = "jnlpba"
 _DISPLAYNAME = "JNLPBA"
 
 _DESCRIPTION = """\
-NER For Bio-Entities
+The data came from the GENIA version 3.02 corpus (Kim et al., 2003).
+This was formed from a controlled search on MEDLINE using the MeSH terms human, blood cells and transcription factors.
+From this search 2,000 abstracts were selected and hand annotated according to a small taxonomy of 48 classes based on
+a chemical classification. Among the classes, 36 terminal classes were used to annotate the GENIA corpus.
 """
 
 _HOMEPAGE = "http://www.geniaproject.org/shared-tasks/bionlp-jnlpba-shared-task-2004"
 
-_LICENSE = 'Creative Commons Attribution 3.0 Unported'
+_LICENSE = "CC_BY_3p0"
```

Comment:
Not sure this is correct. The data archives contain a LICENSE file with "GENIA Project License for Annotated Corpora".